### ISSUE TITLE
Release 1.15.0-rc2 contains conflicts with another lib

### ISSUE THREAD

**Issue by Aleksandr04** [testcontainers/testcontainers-java#3341]
_Fixed in #3159 and released in 1.15.0-rc2_

There are conflicts with **org.jetbrains.annotations** lib. Version: **20.1.0**

![image](https://user-images.githubusercontent.com/17809594/95830135-e7eb7280-0d3f-11eb-8c92-a7ce0ef4d22d.png)

After move back version of testcontainers to 1.14.3 - everything is ok.

**Comment by bsideup** [comment]
It does not seem to "conflict". The library is now `compileOnly`, which is absolutely valid since, on JVM, the class will load fine even if some of the annotations are missing on classpath.

**Comment by Aleksandr04** [comment]
Thanks for the fast support. 
I've made some workaround for the jetbrains lib, but now I catch the following error with initialization:
![image](https://user-images.githubusercontent.com/17809594/95832659-423a0280-0d43-11eb-8361-e3614efa6d08.png)
![image](https://user-images.githubusercontent.com/17809594/95832678-4cf49780-0d43-11eb-976c-50a27fe0068d.png)

Full error message: 

> Caused by: java.lang.IllegalStateException: Failed to verify that image 'mdillon/postgis:10' is a compatible substitute for 'postgres'. This generally means that you are trying to use an image that Testcontainers has not been designed to use. If this is deliberate, and if you are confident that the image is compatible, you should declare compatibility in code using the `asCompatibleSubstituteFor` method. For example:
   DockerImageName myImage = DockerImageName.parse("mdillon/postgis:10").asCompatibleSubstituteFor("postgres");
and then use `myImage` instead.

I tried to use DockerImageName.parse(), but after that, I can't use variables from PostgreSQLContainer, and also it provides more code changes that I expected.

Is it expected? 
One more additional details, I tried to downgrade the version of testcontainers to 1.14.0 and 1.14.2 but catch the following exception: 

> Caused by: org.testcontainers.containers.ContainerFetchException: Can't get Docker image: RemoteDockerImage(imageName=mdillon/postgis:10, imagePullPolicy=DefaultPullPolicy())
Caused by: java.lang.IllegalStateException: Can not connect to Ryuk

(if it will be useful - I use Ubuntu 20.04)
